### ISSUE TITLE
Error when updating story points to jira ticket during sprint poker

### ISSUE THREAD

**Issue by treyparabol** [ParabolInc/parabol#5508]
## Issue - Bug

A [user](https://app.hubspot.com/contacts/3888472/contact/46060451/) 🔒 writes:

> I’ve encountered error when trying to update the story points to jira ticket during sprint poker. We did have story point field and I also granted the permission.
> ![github1](https://user-images.githubusercontent.com/88459947/136975204-094c3349-562b-4ccb-ab5a-a309a40ee7bb.jpg)
> 
> But I still saw this error below:
> ![github2](https://user-images.githubusercontent.com/88459947/136975300-8478457e-da89-4cd2-bd25-1ca8272e0363.png)
> 
> Please let me know what should I do to resolve this.

**Comment by igorlesnenko** [comment]
So I noticed a couple of cases where we can't fix the field:

**1. Internal Server Error**

Happens to me in the following scenario:

- The field has a scope with **`"type": "PROJECT"`** project, for example:

```
{
        "id": "customfield_10035",
        "key": "customfield_10035",
        "name": "Love amount",
        "untranslatedName": "Love amount",
        "custom": true,
        "orderable": true,
        "navigable": true,
        "searchable": true,
        "clauseNames": [
            "cf[10035]",
            "Love amount",
            "Love amount[Short text]"
        ],
        **"scope": {
            "type": "PROJECT",
            "project": {
                "id": "10009"
            }
        },**
        "schema": {
            "type": "string",
            "custom": "com.atlassian.jira.plugin.system.customfieldtypes:textfield",
            "customId": 10035
        }
    }
```

- The screen exists, but screen tabs can't be fetched for some reason:

```
{
      id: 10028,
      name: 'IGORSIMPLE: Simple Default Issue Screen',
      description: '',
      scope: [Object]
    },
```

When fetching tabs:

```
Error: Screen with id 10028 does not exist
```

The above issue should be fixed by #5825, as we will show only fields that exist for the issue.

**2. When you just created Company-managed project (not simplified) on Jira.**

The API still returns that `Story points` field is available for the issue, however, it is not possible to simply update it.

In this case, Fix field works just fine. I checked both on `localhost` and `production` — we request the correct scope every time, and I see no `app does not have scopes` error. 
It seems that it is now fixed.

One thing I additionally did here is the following issue: "Jira - poker meeting does not suggest to fix a field when clicking on a mini estimation card #5846"
As It was very confusing and took some time for me to figure out why I don't see this fix dialog at all.

So this PR should fix it all: #5846

**Additional notes**

1. In the first case I also saw timeout issues, however, I don't see any timeouts when there are no internal server errors, even if we already have a lot of projects on test. 

2. Hiding field like this does not remove it from anywhere:
![image](https://user-images.githubusercontent.com/466991/149941497-e0227464-c1b5-4171-af30-54410c5ef5ec.png)

To remove a field from the screen go to `Project settings -> Screens`

![image](https://user-images.githubusercontent.com/466991/149941661-13d1d626-0fad-4c20-9f68-1005f5f62a1a.png)
